### ISSUE TITLE
Feature Request: Let Us Resize Flex Items with the Mouse

### ISSUE THREAD

**Issue by i-m-abbhay** [prazzon/Flexbox-Labs#9]
## Description
Flexbox Labs is super helpful for learning and designing layouts with CSS Flexbox. Right now, we can add and remove items or change their settings, but we can't drag and resize the items directly using the mouse. This feature would make it easier to use and more fun!

## What We Want
Let users click and drag the edges or corners of a flex item to resize it. When we resize the item, the app should update the width or size of the item instantly so we can see how it looks right away.

## Why This is Useful
- **Easy to Use:** Dragging to resize is faster and simpler than typing values.
- **Saves Time:** It helps us adjust layouts much quicker.
- **Learn Better:** Seeing how the item changes as you drag helps us understand how Flexbox works.

## Things to Think About
- Make sure it works on phones and tablets too.
- Keep it easy to use for everyone, even if they use a keyboard or other tools.
- Don’t let this feature mess up the other cool features like editing properties or selecting items.

Thanks for considering this feature! It would make Flexbox Labs even better for beginners.

**Comment by i-m-abbhay** [comment]
Sorry just now realized there was already a feature request for this.

**Comment by prazzon** [comment]
Duplicate request, but I'm looking into implementing this. Thanks!

**Comment by prazzon** [comment]
I just realized this request is different from the other one. This feature involves resizing the individual items, whereas the other focuses on resizing the playground container.

**Comment by i-m-abbhay** [comment]
Yup, I also got confused and thought this is a duplicate feature request. Just now reread the previous similar requests those are talking about the container size while I was thinking of getting resizable items.

**Comment by Badboyboom** [comment]
CONTRIBUTING.md

**Comment by jitinmaher** [comment]
Is there a plan to build this? I am super interested in consuming this! Great work guys, and thank you for an amazing library!

**Comment by prazzon** [comment]
I'll be looking into implementing this feature in the coming days. If you have any other thoughts on how it should be implemented, I’d love to hear your feedback.
